src/gradata/benchmarks/swe_bench.py

```
@@ -264,11 +264,11 @@ def _load_dataset(dataset_name: str, split: str = "test") -> list[SWEInstance]:
     """
     try:
         from datasets import load_dataset
-    except ImportError:
+    except ImportError as e:
         raise ImportError(
             "SWE-bench data loading requires the 'datasets' package.\n"
             "Install with: pip install datasets"
-        )
+        ) from e
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)